src/Folder.jsx

```diff
@@ -1,28 +1,104 @@
+import { useState, useEffect } from 'react';
+
+import { getFolderList, getLink, getLinkDetail } from './apis/api';
+import shareImg from './assets/folder/share.svg';
+import renameImg from './assets/folder/pen.svg';
+import deleteImg from './assets/folder/delete.svg';
+
+import FolderAddBtn from './components/FolderAddBtn';
 import SearchBar from './components/SearchBar';
 import Card from './components/Card';
 import './Folder.css';
 
-const Folder = ({items}) => {
-  if(!items || !items.links) {
-    return (
-      <div>파트 1에서 만들었던 랜딩 페이지 적용</div>
-    )
+const FOLDER_EDITS = Object.freeze([
+  { src: shareImg, name: "공유" },
+  { src: renameImg, name: "이름변경" },
+  { src: deleteImg, name: "삭제" },
+]);
+
+const Folder = () => {
+  const [folderLists, setFolderLists] = useState([]); // 이건 부모 컴포넌트에서 넘겨주는 게 좋을 듯
+  const [folders, setFolders] = useState([]);
+  const [curfolderList, setCurFolderList] = useState();
```

Comment:
typo가 있는데 VSCode Extension중에서 `Code Spell Checker`를 사용해보시면 좋을 것 같아요.
아래와 같이 밑줄이 뜨게 됩니다.
<img width="752" alt="image" src="https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/51105841/2a61830c-2fa5-484b-abf7-f4cacde053ad">

```suggestion
  const [curFolderList, setCurFolderList] = useState();
```

Reply:
currentFolderName으로 수정하고, 초기화를 진행했습니다.